src/cos_integration.py

```diff
@@ -54,13 +57,10 @@ def _create_scrape_job(self, config: JobConfig, node_name: str, token: str) -> d
             "static_configs": [
                 {
                     "targets": [config.target],
-                    "labels": {"node": node_name},
+                    "labels": {"node": node_name, "cluster": self.charm.model.name},
```

Comment:
it feels like this should be the [`cluster_name`](https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/blob/a41b8d7a7babace4f01d70910af3bfee92fc5465/src/charm.py#L346)

Reply:
nice!